src/features/notifications/NotificationsScreen.tsx

```diff
@@ -44,7 +44,9 @@ const NotificationsScreen = () => {
           refreshing={markNotificationStatus === 'pending'}
         />
       )}
-      {notifications.length === 0 && <EmptyNotifications />}
+      {notifications.length === 0 && markNotificationStatus !== 'pending' && (
```

Comment:
This will keep the `EmptyNotifications` view from showing, but won't show an `ActivityIndicator`, right?

Reply:
Oh good point need one for the loading above too so it shows refreshing